### PR TITLE
Updated installation instructions + arbitrary action spaces in README

### PR DESCRIPTION
**Pull request by jordis-ai2**

### CODE REVIEW

File: docs/installation/installation-allenact.md [hunk 1]
```diff
@@ -16,23 +9,56 @@ PyTorch that supports your CUDA version. In such a case, after installing the be
...
+There are three main installation paths regarding whether you are interested in installing
+
+1. Just the main `allenact` framework (without any plugin).
+1. The main framework and one or more of the available plugins.
+1. The entire library, e.g. for development of more advanced features like a new training algorithm.
+
+#### Standalone framework
+
+```bash
+pip install allenact
```
Comment (Lucaweihs) [inline docs/installation/installation-allenact.md:27]: 👍  we'll need to get allenact on pypi before this works of course. Maybe also add a section about pip installing from GitHub for those people who want to install the bleeding edge version on master without cloning the repo locally.

**Review by Lucaweihs — COMMENTED** [review]
One minor comment but LGTM.

File: docs/installation/installation-allenact.md (hunk 1 above)
Comment (jordis-ai2) [reply docs/installation/installation-allenact.md:27]: Sure. I also realized I had a missing dependency for box2d (gym_plugin). I'll also add it to this PR.

**Comment by jordis-ai2** [comment]
Please double check everything is in order before merging...

**Pull request merged**